packages/lib/src/createUniqueClassName.ts

```diff
@@ -0,0 +1,25 @@
+// https://stackoverflow.com/questions/105034/how-to-create-a-guid-uuid
+// This is a RFC4122 version 4 compliant solution that solves that issue by offsetting the first 13 hex numbers by a hex portion of the timestamp, and once depleted offsets by a hex portion of the microseconds since pageload.
+// That way, even if Math.random is on the same seed, both clients would have to generate the UUID the exact same number of microseconds since pageload (if high-perfomance time is supported) AND at the exact same millisecond (or 10,000+ years later) to get the same UUID:
+
+const generateUUID = () => {
+  let date1 = new Date().getTime(),
+    date2 = (performance && performance.now && performance.now() * 1000) || 0;
+  return 'xxxxxxxx'.replace(/[xy]/g, char => {
```

Comment:
What's faster: `str.replace` or `Array.map`?
Fairly trivial, but I would suggest using `new Array(8).fill('x')` to generate the initial string so it's a bit clearer how many characters we're using

Reply:
I'd guess `replace` too - not to mention the memory implications of using Array. I don't think changing this would have a noticeable impact though, so I'm inclined to keep this as is.

Reply:
In that case, would the regex just be `/x/g`, since we're not using the `y` from the original SO post?